packager/media/formats/webm/webm_muxer.cc

```diff
@@ -47,7 +47,8 @@ Status WebMMuxer::InitializeMuxer() {
 }
 
 Status WebMMuxer::Finalize() {
-  DCHECK(segmenter_);
+  if (!segmenter_)
```

Comment:
That happens only when Finalize is called without calling Initialize.

@murillo128 Have you checked why it happens?

Reply:
This seems to be happening when no media is received during the duration of the segmenter.

I am using shaka packager integrated into my media server to record media coming via streaming, so it may happen (due to streaming client malfunction, network disruption or reconnections) that there are periods of time in which media is not received.